### ISSUE TITLE
Build wheels for `musllinux`

### ISSUE THREAD

**Issue by mochipon** [awslabs/aws-crt-python#315]
Precompiled wheels for musl based Linux distributions (e.g., alpine linux) are not provided at this moment (ref. #272).
Therefore, it is time-consuming to utilize this library in the container environment that we often see.

`musllinux` spec has just landed at [PEP656](https://www.python.org/dev/peps/pep-0656/) and `musllinux` wheels must be super useful for these environments.

I understand that there are several improvements planned for CI at #312 #308 and I'm sure they will take priority.

**Comment by qqaatw** [comment]
Really needs precompiled wheels for musl based linux as many IoT oriented operating systems are musl based, and there is usually no preinstalled build tools on these systems so the SDK could not be built from source.

**Comment by adolfogc** [comment]
This would also be great for using Alpine-based Docker images on Balena.

**Comment by qqaatw** [comment]
Any update on this?

**Comment by qqaatw** [comment]
I just set up a [repository](https://github.com/qqaatw/aws-crt-python-musllinux) that periodically builds musllinux wheels for awscrt, one can install them through:

```
pip install awscrt -f https://qqaatw.github.io/aws-crt-python-musllinux/
```

**Comment by graebm** [comment]
musllinux wheels are available as of version [0.16.21](https://github.com/awslabs/aws-crt-python/releases/tag/v0.16.21)
